### PR TITLE
[INLONG-5689][Manager] PulsarSource set fieldDelimiter when use CSV format

### LINKED ISSUE #5689: [Feature][Manager] PulsarSource support set fieldDelimiter when use CSV format
### Description

PulsarSource set fieldDelimiter when use CSV format

### Use case

none

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by e-mhui**
### Prepare a Pull Request

[INLONG-5689][Manager] PulsarSource set fieldDelimiter when use CSV format

- Fixes #5689

### Motivation

PulsarSource set fieldDelimiter when use CSV format

### CODE REVIEW

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java [hunk 1]
```diff
@@ -66,6 +66,8 @@ public class InlongConstants {
 
     public static final String PULSAR_QUEUE_TYPE_PARALLEL = "PARALLEL";
 
+    public static final String PULSAR_CSV_FIELDDELIMITER = "fieldDelimiter";
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java:69]: Suggested changing to `FIELD_DELIMITER`.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/pulsar/PulsarSourceOperator.java [hunk 2]
```diff
@@ -142,6 +142,10 @@ public Map<String, List<StreamSource>> getSourcesMap(InlongGroupInfo groupInfo,
                         && StringUtils.isNotEmpty(sourceInfo.getSerializationType())) {
                     pulsarSource.setSerializationType(sourceInfo.getSerializationType());
                 }
+                if (DataTypeEnum.CSV.getName().equalsIgnoreCase(pulsarSource.getSerializationType())) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/pulsar/PulsarSourceOperator.java:145]: Suggested move to the out of `for` loop.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java (hunk 1 above)
Comment (e-mhui) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java:69]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/pulsar/PulsarSourceOperator.java (hunk 2 above)
Comment (e-mhui) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/pulsar/PulsarSourceOperator.java]: done

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**